.changeset/beige-glasses-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/ui-react-liveness': patch
+'@aws-amplify/ui': patch
+---
+
+Losslessly Compress Images
```

Comment:
```suggestion
Losslessly compress images
```

just to be consistent with https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CHANGELOG.md